testsuite/run_sets/github_validation/github_validation_init_clients.yml

```diff
@@ -6,4 +6,5 @@
 - features/github_validation/init_clients/sle_minion.feature
```

Comment:
I think that we need to adapt `sle_ssh_minion.feature` in order to bootstrap using the proxy.

Reply:
the same https://github.com/uyuni-project/uyuni/pull/8446#discussion_r1553578751